content/docs/command-reference/init.md

```diff
@@ -32,88 +33,92 @@ that are hidden from the user. This directory is automatically staged with
+DVC in the Git repo root:
 
-Let's imagine we have an existing Git repository that is split into sub-projects
-(monorepo). In this case `dvc init --subdir` can be run in one or many
-sub-projects to mitigate the issues of initializing in the Git repository root:
+- Repository maintainers might not allow a top level `.dvc/` directory,
+  especially if DVC is being used by several sub-projects (monorepo).
 
-- Repository maintainers might not allow extra `.dvc` top level directory,
-  especially if DVC is being used by a small number of sub-projects.
+- DVC config file, cache directory,
+  [etc.](/doc/user-guide/dvc-files-and-directories) are shared across different
+  sub-projects. This makes it difficult to use different DVC settings,
+  [remote storage](/doc/command-reference/remote) locations, etc.
 
-- Not enough isolation/granularity - DVC config, cache, and other files are
-  shared across different sub-projects. Means that it's not easy to use
-  different remote storages, for example, for different sub-projects, etc.
-
-- Not enough isolation/granularity - commands like `dvc pull`, `dvc checkout`,
-  and others analyze the whole repository to look for `dvc.yaml` or `.dvc` files
-  to download files and directories, to reproduce <abbr>pipelines</abbr>, etc.
-  It can be expensive in the large repositories with a lot of projects.
-
-- Not enough isolation/granularity - commands like `dvc metrics diff`, `dvc dag`
-  and others by default dump all the metrics, all the pipelines, etc.
+- Many DVC commands can explore the whole <abbr>DVC repository</abbr> to find
+  DVC-tracked data and pipelines to work with. This can be undesirable and
+  inefficient for large monorepos.
 
 #### How does it affect DVC commands?
 
-No matter what mode is used, DVC looks for the `.dvc` directory when it starts
-(from the current working directory and up). Location of the found `.dvc`
-directory determines the root of the DVC project. (In case of `--subdir` it
-might happen that Git repository root is located at different path than the DVC
-project root.)
+The <abbr>project</abbr> root is found by DVC by looking for `.dvc/` from the
+current working directory, up. It defines the scope of action for most DVC
+commands (e.g. `dvc repro`, `dvc pull`, `dvc metrics diff`), meaning that only
+`dvc.yaml`, `.dvc` files, etc. inside the project are usable by the commands.
 
-DVC project root defines the scope for most DVC commands. Mostly meaning that
-all `dvc.yaml` and `.dvc` files under the root path are being analyzed.
+With `--subdir`, the project root will be found before the Git root, making sure
+the scope of DVC commands run here is constrained to this project alone, even if
+there are more DVC-related files elsewhere in the repo. Similarly, DVC commands
+run outside this project root will ignore its contents.
 
-If there are multiple DVC sub-projects but they _are not_ nested, e.g.:
+**Simple structures**: multiple `--subdir` projects, not nested, e.g.:
```

Comment:
I would not call these simple or advanced. Nesting alone does not make them simple or advanced.

Reply:
Renamed to "Nested".

Reply:
OK, removed both "Simple" and "Nested" now (in 5b47bd4f).